gatsby/content/blog/2021/04/2021-04-15-old-gitter-bridge-eol.mdx

```diff
@@ -0,0 +1,26 @@
+---
+  date: '2021-04-15'
+  title: Old Gitter bridge end of life (2021-04-21) - to be replaced with native bridge
+  categories:
+    - bridge
+  author: Eric Eastwood
```

Comment:
@Half-Shot Would this come across better from you? (no preference is fine too)

Reply:
We could say "Bridge Team", but since you wrote it I'm happy for you to announce it personally.

Reply:
`Bridge Team` works for me :+1: